<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomsDialog</class>
 <widget class="QDialog" name="RoomsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Room Management</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Floor 1</string>
   </property>
   <widget class="QPushButton" name="pushButton101">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>101</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton102">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>102</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton103">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>103</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Floor 2</string>
   </property>
   <widget class="QPushButton" name="pushButton201">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>201</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton202">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>202 (Suite)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
